### PR TITLE
Bug #5  redesign for the view building information popup

### PR DESCRIPTION
**Pull request by Lilloush**
## Description:
Fully updated the design of the Building information popup as follows:

- changed description to include the address of the building and the department it belongs to 
- changed the get directions, indoor map and save buttons to clickable Icons
- added different facilities as Icons (coffee, restaurants, zen dens, parking, metro access)
- Learn more button became More button which now leads to the web page of the building

## Related User Story:
#139 

## Screenshot/Demo:
image

image

## How to test:
N/A

## Type of Change:
- [ ] Feature
- [x] Bug fix
- [ ] Refactor
- [ ] Documentation update
- [ ] Tests
- [ ] Others

## Pull Request Checklist
- [x] Flutter CI test passed
- [x] Sonar test passed
- [x] Codecov test passed
- [x] Application runs without errors
- [x] Feature tested locally on a high level
- [x] Architecture conventions respected
- [x] No API keys or secrets committed
- [x] Document when necessary

## PR Review Checklist (dev)
- [x] Code is readable and well structured
- [x] No obvious bugs or edge cases missed
- [x] Tests are adequate
- [x] Naming and organization are clear
- [x] Feature works as described
- [x] No unnecessary complexity
- [x] Follows project architecture

## PR Review Checklist (QA)
- [x] Made sure to test the application fully
- [x] No bugs created by this PR

### CODE REVIEW

File: lib/shared/widgets/building_info_popup.dart [hunk 1]
```diff
@@ -41,13 +123,30 @@ class BuildingInfoPopup extends StatelessWidget {
...
-                const Icon(Icons.accessible, size: 18, color: burgundy),
-                const Spacer(),
+                _buildIconButton(
+                  icon: _isSaved ? Icons.bookmark : Icons.bookmark_border,
+                  tooltip: _isSaved ? 'Unsave' : 'Save',
+                  onPressed: () {
+                    setState(() {
+                      _isSaved = !_isSaved;
+                    });
+                  },
```
Comment (enyc24) [inline lib/shared/widgets/building_info_popup.dart]: The saved icon should also have an if statement. This is a functionality that is only available if you are a user that is logged in. Right now, if I continue as guest, I still have the option of saving.

File: lib/shared/widgets/building_info_popup.dart [hunk 2]
```diff
@@ -1,28 +1,112 @@
...
+      message: tooltip,
+      child: IconButton(
+        onPressed: onPressed,
+        icon: Icon(
+          icon,
+          color: burgundy,
+        ),
+        iconSize: 25,
+        padding: EdgeInsets.zero,
+        constraints: const BoxConstraints.tightFor(width: 32, height: 32),
```
Comment (enyc24) [inline lib/shared/widgets/building_info_popup.dart]: The pop up icon is cut off on the screen, can we fix the widget to be in the center please. This happens for all buildings. 

Image

**Comment by Lilloush** [comment]
No it's for the Zen dens
